### PR TITLE
add ability to build in debug

### PR DESCRIPTION
**Pull request by briandowns**
### What's Changed:

Adds the ability to build in debug to the buildAndInstall script.

### Type of Change:

- [ ] Bug fix
- [ X ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#

### Housekeeping:

- [ ] Tests have been updated to reflect the changes done within this PR (if applicable).
- [ ] Documentation has been updated to reflect the changes done within this PR (if applicable).

#

### CODE REVIEW

File: ops/buildAndInstallLatest.du [hunk 1]
```diff
@@ -25,10 +25,21 @@ def pullLatest(isFork) {
 }
 
 // build builds the latest code pulled.
-def build() {
+def build(debug) {
+    var buildType = "Release";
+    if (debug) {
+        buildType = "Debug";
+    }
```
Comment (Jason2605) [inline ops/buildAndInstallLatest.du:32]:
```suggestion
    const buildType = debug ? "Debug" : "Release";
```

Comment (Jason2605) [reply ops/buildAndInstallLatest.du:32]: Could also potentially Enum those up as well but meh

**Review by Jason2605 — APPROVED** [review]
Nitpick so I've approved anyways - Thanks for this!

**Comment by briandowns** [comment]
I completely forgot Dictu has ternary. I have a few places in my other projects I can update. Thanks!

**Comment by Jason2605** [comment]
Honestly I forget half of the stuff as well haha, thanks for updating!

**Pull request merged**